src/graphx/graphx.asm

```diff
@@ -523,13 +525,15 @@ gfx_GetPixel:
 	inc	hl
 	inc	hl
 	inc	hl			; move to next argument
-	ld	de,0
 	ld	e,(hl)			; e = y coordinate
-	call	_PixelPtr
```

Comment:
I believe this routine does some checks first, here is it safe to get rid of that?

Reply:
My suggestion : not verifying x,y coordinates as the result is just a value of one pixel on screen. Until now, If coord outside of screen, what's the result ?

Reply:
when coordinates are definitely outside, then what's the returned value for color , please ?

Reply:
to check or not to check ?..That's the question :)

Reply:
It seems a little sketchy that this could read from an arbitrary memory address, but jacobly seems to think that should be okay.